README.md

```diff
@@ -81,6 +81,44 @@ end
 `text` fields will be full-text searchable. Other fields (e.g.,
 `integer` and `string`) can be used to scope queries.
 
+### Nested Documents support
+
+You can use the `child_documents` feature to add nested documents to other models,
+as such:
+
+```ruby
+class Comment < ActiveRecord::Base
+  # NOTE: this is not necessary, Solr uses '_root_' field to refer to the parent document
```

Comment:
I think you missed `searchable` declaration here

Reply:
Fixed, sorry about that